chrome-extension/background/utils/processSteps.js

```diff
@@ -0,0 +1,97 @@
+/*global chrome*/
+export const processSteps = async (data) => {
+  const steps = data.steps;
+  const server_url = "http://localhost:8000"; // Cant have env variables in chrome extensions
+
+  const active_tab = await chrome.tabs.query({
+    active: true,
+    currentWindow: true,
+  });
+
+  await chrome.scripting.executeScript({
+    target: { tabId: active_tab[0].id },
+    files: ["background/utils/executeActionsScript.js"],
+  });
+
+  let init_time = null;
+  for (const step of steps) {
+    init_time = new Date().getTime();
+    step.state = "in-progress";
+
+    const image = await chrome.tabs.captureVisibleTab();
+    const data = {
+      instruction: {
+        ...step.instruction,
+      },
+      currentView: image,
```

Comment:
the API uses snake case. changing this to camel case breaks the call
```suggestion
      current_view: image,
```